modules/monitoring-modifying-retention-time-and-size-for-prometheus-metrics-data.adoc

```diff
@@ -0,0 +1,134 @@
+// Module included in the following assemblies:
+//
+// * monitoring/configuring-the-monitoring-stack.adoc
+
+:_content-type: PROCEDURE
+[id="modifying-retention-time-and-size-for-prometheus-metrics-data_{context}"]
+= Modifying the retention time and size for Prometheus metrics data
+
+By default, Prometheus automatically retains metrics data for 15 days.
+You can modify the retention time to change how soon data is deleted.
+You can also configure the maximum amount of disk space used by the retained metrics data.
+If this size limit is reached, the oldest data is deleted first until the disk space used by the data is again below the maximum.
+
+Note the following behaviors of these data retention settings:
+
+* The retention settings apply only to data block directories. 
+The settings are not used for data located in the `wal` and `chunks_head` directories.
+* The retention settings are used only when Prometheus writes a new data block to a data block directory, which occurs every two hours and after the write-ahead log (WAL) has recorded at least three hours of data.
+* If you do not explicitly define values for either the retention time or retention size fields, retention time defaults to 15 days, and retention size is not set.
+* If you define values for both retention time and retention size fields, both values apply.
+If any data blocks exceed either the defined retention time or the defined size limit, then these data blocks are purged.
+* If you define a value for the retention size field and do not define the retention time field, only the retention size value applies.
+* If you do not define a value for the retention size field and only define a value for the retention time field, only the retention time value applies
+
```

Comment:
Should we also add how both retention and retentionSize  settings works w.r.t to cmo?

When neither retention nor retentionSize are defined, a default of 15d for time retention applies.
When "retentionSize" is defined and "retention" isn't, only size retention applies.
When "retentionSize" isn't defined and "retention" is, only time retention applies 
When both time and size retention fields are defined, both apply. If any of them is true at some point in time (either blocks are too old or the size of all blocks exceeds the size limit) then data is purged.

Reply:
Yes. We should add these. I will do so.

Reply:
Added new content to account for these scenarios